### PR TITLE
cpp fib autograder

### PR DESCRIPTION
**Pull request by Huahan98**

### CODE REVIEW

File: Cpp-fibonacci/autograde-Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+all:
+	tar xvf autograder.tar
```
Comment (xinyis991105) [inline Cpp-fibonacci/autograde-Makefile]: autograder.tar -> autograde.tar

File: Cpp-fibonacci/autograde-Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	cp fib.cpp test_autograder
+	
+	(cd test_autograder; make clean; make)
```
Comment (xinyis991105) [inline Cpp-fibonacci/autograde-Makefile:5]: See the autograding feedback at https://autolab-dev.andrew.cmu.edu/courses/15213-s21/assessments/cppfiblab/viewFeedback?feedback=510&submission_id=2103. I think calling a second make leaves the scoring JSON overridden with Makefile's leaving directory statement. Can you find a way around this? Maybe for the inner Makefile just compile, and then call the test program from the outer Makefile?

File: Cpp-fibonacci/autograde-Makefile (hunk 1 above)
Comment (Huahan98) [reply Cpp-fibonacci/autograde-Makefile]: fixed

**Review by xinyis991105 — CHANGES_REQUESTED** [review]
See my other comment for file autograde-Makefile

**Review by xinyis991105 — APPROVED** [review]
LGTM! Good job!

**Pull request merged**